src/Model/Volunteer.hs

```diff
@@ -0,0 +1,19 @@
+module Model.Volunteer where
+
+import Import
+
+import Control.Monad.Trans.Writer.Strict (tell)
+
+fetchAllInterestsDB :: DB [Entity Interest]
+fetchAllInterestsDB = selectList [] []
+{-# INLINE fetchAllInterestsDB #-}
```

Comment:
Fixed, but can we stop bikeshedding now? You can open up any module and see a hundred things that could be written cleaner